charts/digital-twin-bundle/values.yaml

```diff
@@ -0,0 +1,40 @@
+# #############################################################################
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# #############################################################################
+
+digital-twin-registry:
+  nameOverride: dataprovider-dtr
+  enablePostgres: false
+  registry:
+    host: dataprovider-dtr.test
+    dataSource:
+      url: "jdbc:postgresql://digital-twin-digital-twin-registry-db:5432/dtr"
```

Comment:
Here is still an issue if the helm install name is not "digital-twin". Since dtr does not allow templating for this value we might need to hardcode this or set it via the "--set"  property when installing. 
e.g.: `helm install <release-name> charts/digital-twin-bundle --set digital-twin-registry.registry.dataSource.url=jdbc:postgresql://<release-name>-digital-twin-registry-db:5432/dtr`